src/lv_core/lv_indev.c

```diff
@@ -669,7 +669,7 @@ static void indev_encoder_proc(lv_indev_t * i, lv_indev_data_t * data)
                 /*On enter long press toggle edit mode.*/
                 if(editable) {
                     /*Don't leave edit mode if there is only one object (nowhere to navigate)*/
-                    if(_lv_ll_is_empty(&g->obj_ll) == false) {
+                    if(_lv_ll_is_empty(&g->obj_ll) == false && _lv_ll_get_len(&g->obj_ll) > 1 ) {
```

Comment:
I think it should be simply:
```suggestion
                    if (_lv_ll_get_len(&g->obj_ll) > 1 ) {
```